server/pkg/webserver/base/scheme.go

```diff
@@ -0,0 +1,75 @@
+package base
```

Comment:
You can ref https://github.com/vesoft-inc/nebula-dashboard-ent/tree/master/backend/pkg/webserver .